src/api/app/views/webui/comment/_comment_field.html.haml

```diff
@@ -1,7 +1,7 @@
-= form_for(comment, action: 'create', method: :post, remote: true, html: { class: 'new-comment-form' }) do |f|
+= form_for(comment, method: form_method, remote: true, html: { class: "#{form_method}-comment-form" }) do |f|
   = hidden_field_tag :commentable_type, commentable.class.name
   = hidden_field_tag :commentable_id, commentable.id
   = f.hidden_field :parent_id, value: comment.parent_id
   ~ f.text_area :body, rows: '4', placeholder: 'Add a new comment (markdown markup supported)', required: true,
     class: 'w-100 mb-3 form-control comment-field'
-  = f.submit 'Add comment', class: 'btn btn-primary', data: { disable_with: 'Creating comment...' }
+  = f.submit action_label, class: 'btn btn-primary', data: { disable_with: 'Creating comment...' }
```

Comment:
Both the button name and the `data-disable_with` attribute depend on the context in which this partial is rendered. We can rely on the `form_method` local variable to set the button name and the data attribute. This should be done in this partial so we don't have to always redefine this whenever using the partial.